packages/polyfills/streamToPromise.test.ts

```diff
@@ -0,0 +1,53 @@
+import { beforeEach, describe, expect, test } from 'vitest';
+
+import Array from './Array.fromAsync';
+
+// eslint-disable-next-line @typescript-eslint/unbound-method
+const streamToPromise = Array.fromAsync;
+
+describe('streamToPromise()', () => {
```

Comment:
Hm, I feel like these tests are no longer needed — even the ones below under `old streamToPromise behavior`. You're essentially testing something that's part of the Chrome JavaScript API, which should already be well-tested and guaranteed to work. No need to retest it ourselves.

If anything, if there are places where we're using `Array.fromAsync` that _does_ depend on old `streamToPromise` behavior, we should write tests to make sure it's accounted for the fact that `Array.fromAsync` works differently.

Reply:
maintaining the old tests was useful to demonstrate the api satisfies the behavior we needed